src/state/entities/entitiesConfigManager.ts

```diff
@@ -133,4 +136,60 @@ entitiesConfigManager.register({
     },
 });
 
+entitiesConfigManager.register({
+    asyncEntityKey: ASYNC_ENTITY_KEYS.connectionTypes,
+    operationsConfig: {
+        fetch: {
+            api: api.constants.fetchConnectionTypes,
+        },
+    },
+});
+
+entitiesConfigManager.register({
+    asyncEntityKey: ASYNC_ENTITY_KEYS.componentTypes,
+    operationsConfig: {
+        fetch: {
+            api: api.constants.fetchComponentTypes,
+        },
+    },
+});
+
+entitiesConfigManager.register({
+    asyncEntityKey: ASYNC_ENTITY_KEYS.openapi,
+    operationsConfig: {
+        create: {
+            api: api.openapi.transformDocumentation,
+            apiInputSelector: ({ extraInput }) => extraInput as IOpenAPI,
+        },
+    },
+});
+
+entitiesConfigManager.register({
+    asyncEntityKey: ASYNC_ENTITY_KEYS.connections,
+    operationsConfig: {
+        create: {
+            api: api.connections.createConnection,
+            apiInputSelector: ({ extraInput }) => extraInput as IConnectionEntity,
+        },
+        update: {
+            api: api.connections.updateConnection,
+            apiInputSelector: ({ extraInput }) => extraInput as IConnectionEntity,
+        },
+    },
+});
+
+entitiesConfigManager.register({
+    asyncEntityKey: ASYNC_ENTITY_KEYS.components,
+    operationsConfig: {
+        create: {
+            api: api.components.createComponent,
+            apiInputSelector: ({ extraInput }) => extraInput as IComponentEntity,
+        },
+        update: {
+            api: api.components.updateComponent,
+            apiInputSelector: ({ extraInput }) => extraInput as IComponentEntity,
+        },
+    },
+});
```

Comment:
Be cautious with placing these connections and components in the state manager. These OpenAPI components and connections should not be confused with your current issue of the 'global' components